ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -115,6 +115,65 @@ presubmits:
           name: prow-job-cluster-launch-installer-e2e
         name: job-definition
     trigger: ((?m)^/test( all| e2e-aws-all),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-openstack
+    decorate: true
+    name: pull-ci-openshift-installer-master-e2e-openstack
+    optional: true
```

Comment:
I'd like to make this job optional for now but I'm not sure whether this is the right way to do it. I don't want this job to block patches until it's stable.

Reply:
That's the appropriate way to do it